docroot/themes/humsci/humsci_basic/humsci_basic.theme

```diff
@@ -228,6 +228,50 @@ function humsci_basic_preprocess_pattern_vertical_link_card(&$variables) {
   }
 }
 
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function humsci_basic_preprocess_pattern_vertical_link_card(&$variables) {
+  _humsci_basic_make_title_the_link($variables);
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function humsci_basic_preprocess_pattern_vertical_card(&$variables) {
+  $context = $variables['context'];
+  if ($context->isOfType('layout')) {
+    // Functionality for paragraphs
+    _humsci_basic_make_title_the_link($variables);
+  } else {
+    // Functionality for Views.
+    $renderer = \Drupal::service('renderer');
+    $title_rendered = $renderer->renderPlain($variables['title']);
+
+    // Create a new DOMDocument instance and load the rendered title HTML.
+    $dom = new DOMDocument();
+    $dom->loadHTML($title_rendered);
+
+    // Create a DOMXPath instance to query the DOM.
+    $xpath = new DOMXPath($dom);
+
+    // Find all anchor elements in the DOM.
+    $links = $xpath->query('//a');
```

Comment:
Wouldn't a regex be more simple?

Reply:
We're using `DOMDocument` because we need to alter the markup, adding an attribute to the link. We think that doing this with regular expressions / string manipulation is more complex and error prone. Also, we wanted to keep consistency with other similar code used in `_humsci_basic_make_title_the_link()`.

If you still consider that a regex is a better approach, let me know and I can update the code.